src/Microsoft.Identity.Web/MicrosoftIdentityOptions.cs

```diff
@@ -25,6 +25,17 @@ public class MicrosoftIdentityOptions : OpenIdConnectOptions
         /// </summary>
         public string Domain { get; set; }
 
+        /// <summary>
+        /// In a Web app, gets or sets the RedirectUri of (URI where the token will be sent back by
+        /// Azure Active Directory or Azure Active Directory B2C.
+        /// This property is exclusive with <see cref="CallbackPath"/> which should be used preferably if you don't want
+        /// to have a different deployed configuration from your developper configuration.
+        /// There are cases where RedirectUri is needed, for instance when you use a reverse proxy that transforms https
+        /// URLs (external world) to http URLs (inside the protected area). This can also be useful for Web apps running
+        /// in containers (for the same reasons)
+        /// </summary>
```

Comment:
You can probably add a small note on what redirect uri will be used if one is not specified..